src/components/main.tsx

```diff
@@ -68,6 +86,9 @@ const Main = () => {
                             <img src={MetaMask} alt="Connect with Metamask" />
                         </Button>
                     </InputGroupAddon>
+                    {!validAddress && <FormFeedback style={{color:'yellow'}} invalid>
```

Comment:
The basic yellow color doesn't fit the style of the website imho. Please use one of the existing color variables: https://github.com/burnsignal/BrightID-EthereumAddressService/blob/8fbc2d2d9443dc805c25a2f2ddd8349681f8eedd/src/assets/scss/bright-ethereum/_variables.scss#L1

that fits better.

Also, if the colored text doesn't look good, consider leaving it white and only applying validation to the `FormFeedback` and not the `Input`